confd/etc/calico/confd/templates/bird6_ipam.cfg.template

```diff
@@ -38,6 +38,19 @@ function calico_export_to_bgp_peers(bool internal_peer) {
   if ( net ~ {{$cidr}} ) then { accept; }
   {{- end}}
 {{- end}}
+{{- $is_rr_key := printf "/bgp/v1/host/%s/rr_cluster_id" (getenv "NODENAME")}}
+{{- $lb_ips := "/bgp/v1/global/svc_loadbalancer_ips"}}
+{{- if exists $is_rr_key}}{{$is_rr := getv $is_rr_key}}
+{{- if and (not (eq $is_rr "")) (exists $lb_ips)}}
+
+  {{- range split (getv $lb_ips) ","}}
```

Comment:
I think we should have a comment here that gets rendered in the resulting config files to make them easier to read / debug. Similar to the comment on line 34 ("Export static routes")

e.g.,

```
# Configured as a RR - accept any routes within configured LB service IP ranges
```

Reply:
In the first iterations I added something like that, but noticed that this message will be printed in the IPv6/IPv4 file even if there are no IPV6/IPv4 routes in the `lb_ips` variables as there's no distinct variables for each protocol. I can add it back but would it be a problem?